nexus/bin/nxs-test

```diff
@@ -2257,7 +2232,6 @@ if __name__=='__main__':
     parser = OptionParser(
         usage='usage: %prog [options]',
         add_help_option=False,
-        version='%prog {}.{}.{}'.format(*nexus_version)
```

Comment:
Same as above.

Reply:
Can you show timings or something?  Similar to the comment above, the `versions.py` import should be very cheap and safe.